backend/user_management/routers/product.py

```
@@ -93,7 +90,7 @@ def get_product(product_id: int,db: Session = Depends(get_db)):
 
 
 
-@router.get("/get-all-products", response_model=List[schemas.ProductOut])
+@router.get("/products", response_model=List[schemas.ProductOut])
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)